test/CMakeLists.txt

```diff
@@ -130,6 +133,7 @@ target_link_libraries(test_xtensor_io_lib
     ${CMAKE_DL_LIBS}
     ${GTEST_BOTH_LIBRARIES}
     ${CMAKE_THREAD_LIBS_INIT}
+    storage_client
```

Comment:
This is not required anymore if you move the dependency on GCS to the main CMakeLists.